packager/app/mpd_flags.cc

```diff
@@ -69,3 +69,15 @@ DEFINE_bool(include_mspr_pro_for_playready,
             "If enabled, PlayReady Object <mspr:pro> will be inserted into "
             "<ContentProtection ...> element alongside with <cenc:pssh> "
             "when using PlayReady protection system.");
+DEFINE_bool(dash_force_segment_list,
+            false,
+            "Uses SegmentList instead of SegmentBase. Use this if the "
+            "content is huge and the total number of (sub)segment references "
+            "is greater than what the sidx atom allows (65535). "
+            "Note that this needs to be used with the "
+            "--generate_sidx_in_media_segments options since "
+            "sidx generation needs to be skipped if SegmentList is used "
+            "(the extra references are truncated and the stream would not play "
+            "until the end).");
```

Comment:
Aren't the contents still play with `--generate_sidx_in_media_segments` set? In that case, I prefer removing this texts.